dannymartinm/beam
# Beam 12812 githubactions windows

### What does this PR do?
Updates the "runs-on" tag of java_tests, python_tests and build_wheels.yml in order to execute on self-hosted for Windows, there is also an additional step for build_wheels that cleans the folder wheelhouse, fixing some issues with the existing .whl files.

### Why is this important?
It is necessary  for the final migration in where all the tags will be updated to run on self-hosted.
Where should the reviewer start?
.github/workflows: python_test.yml, java_tests.yml and build_wheels.yml.


### How should this be manually tested?

Recreate a new instance-template by replacing the "windows-runner-repo" and "windows-runner-token" secrets in the gcloud command located inside .github/dockerized-gh-actions-runners/self-hosted-windows.

New secrets must be linked with the test repo.

Wait for the automatic registration process to finish and once you see the self-hosted runners listed in your repo you can launch a workflow that should be now redirected to your self-hosted.

### Any background context you want to provide?
N/A

### What are the relevant tickets?
N/A

### Type of change


### Checklist
